kata-deploy/kata-deploy.yaml

```diff
@@ -127,7 +121,7 @@ spec:
           kata-containers.io/container-runtime: containerd
       containers:
       - name: kube-kata
-        image: katadocker/kata-deploy:1.1.0
```

Comment:
It think is not needed to reference to a tag anymore?

Reply:
yeah, the second commit addresses this.